src/Pos/Binary/Txp.hs

```diff
@@ -68,13 +68,24 @@ instance Bi T.TxDistribution where
                     nonEmpty $ replicate n []
                 Right ds -> pure ds
 
+instance Bi T.TxSigData where
+    put T.TxSigData{..} = do
```

Comment:
> Also, did someone benchmarked difference so I stop worrying about that :)

Nope, I think nobody benchmarked it.

To be honest, since `binary` is the slowest serialization library there is, I don't think spending time on rewriting `Bi` instances is reasonable. If `Bi` isn't a bottleneck, we don't have to do it, and if it is a bottleneck, we should switch to a different library instead of using `binary`.